utils/kubernetes/kompose/convert.go

```diff
@@ -193,6 +193,7 @@ func convert(opt kobject.ConvertOptions) error {
 	if err != nil {
 		return err
 	}
+	fmt.Println(komposeObject)
```

Comment:
Why print the obj?

Reply:
This komposeObject was throwing an error on linting check as it was initialized and was not used anywhere.

Reply:
Maybe we can remove it.

Reply:
sure @Aisuko